### PR TITLE
Fix functional tests

### PR DESCRIPTION
**Pull request by sebastienros**
Saw some exceptions in the functional tests logs. Since these are from event handlers they were not interfering with the page rendering and hence not triggering a test failure. We may want to check if the logs contain and exception to trigger function test failures.

### CODE REVIEW

File: src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Utilities/StringExtensions.cs [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
     {
+        if (string.IsNullOrEmpty(alternateElement))
+        {
+            return "";
+        }
+
+        if (!alternateElement.AsSpan().ContainsAny(AlternateChars))
+        {
+            return alternateElement;
+        }
```
Comment (Piedone) [inline src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Utilities/StringExtensions.cs:24]: How does this fix an exception? Or is this condition just for perf?

**Review by Piedone — COMMENTED** [review]
https://github.com/OrchardCMS/OrchardCore/pull/11194 checks logs as well BTW.

File: src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Utilities/StringExtensions.cs (hunk 1 above)
Comment (sebastienros) [reply src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Utilities/StringExtensions.cs:24]: Right, it's unrelated, just trying to squeeze some perf. Also an opportunity to raise awareness about `SeachValues`.

**Review by Piedone — APPROVED** [review]

File: src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Utilities/StringExtensions.cs (hunk 1 above)
Comment (Piedone) [reply src/OrchardCore/OrchardCore.DisplayManagement.Abstractions/Utilities/StringExtensions.cs:24]: Yeah, that's good, TIL.

**Pull request merged**